### PR TITLE
Re-add the above-the-fold collection display

### PR DESCRIPTION
**Pull request by thatbudakguy**
This was removed in 6fc28fa.

### CODE REVIEW

File: app/views/purl/_mods_upper_metadata_section.html.erb [hunk 1]
```diff
@@ -1,11 +1,14 @@
 <% document_notes = document.mods.note.select { |x| x.label =~ /Preferred citation/i } %>
-<% if document_notes.present? %>
+<% if document_notes.present? || document.mods.collection.any? %>
```
Comment (jcoyne) [inline app/views/purl/_mods_upper_metadata_section.html.erb:2]: Can we combine `fields = document_notes + document.mods.collection` and iterate over that. Both are handled the same way.

**Comment by jcoyne** [comment]
@thatbudakguy won't this display the collections twice on the page?

**Comment by thatbudakguy** [comment]
yeah, it will. I think we're waiting to hear back about whether that was an intentional choice in the design (which I could see it being).

**Comment by astridu** [comment]
@amyehodge Also commented in Slack but will also put this comment in the ticket. It was intentional because we were going to have the collection in the right column, but since we're not doing that, not sure if Amy wants to keep the collection name under the the item title?

**Comment by amyehodge** [comment]
Since we are not currently doing the major redesign with the two columns, everything on the page should stay where it has been, so the collection name should be up above the viewer. @astridu does that mean the new link should be there as well? Or is that going further down the page?

**Comment by astridu** [comment]
@amyehodge The link will be under the Collections section in the body of the page. I put the collections line back at the top.

File: app/views/purl/_mods_upper_metadata_section.html.erb (hunk 1 above)
Comment (thatbudakguy) [reply app/views/purl/_mods_upper_metadata_section.html.erb]: updated

**Review by jcoyne — APPROVED** [review]

**Pull request merged**